frontend/public/components/utils/status-box.tsx

```diff
@@ -75,6 +75,20 @@ export const EmptyBox: React.FC<EmptyBoxProps> = ({ label }) => {
 };
 EmptyBox.displayName = 'EmptyBox';
 
+export const PVsLoading: React.FC<EmptyBoxProps> = ({ label }) => {
+  const { t } = useTranslation();
+  return (
+    <Box>
+      <div data-test="empty-message" className="text-center">
+        {label
+          ? t('ceph-storage-plugin~PVs are being provisioned on {{label}}, please wait', { label })
+          : t('ceph-storage-plugin~PVs are being provisioned, please wait')}
+      </div>
+    </Box>
+  );
+};
+PVsLoading.displayName = 'PVsLoading';
+
```

Comment:
I still see this in the public component, can you move this to ceph plugin.

Reply:
Moved the `PVsLoading` component into `ceph-storage-plugin/src/components/ocs-install/pvs-available-capacity.tsx` file.
@cloudbehl , please take a look...